### PR TITLE
Update standalone.md

### PR DESCRIPTION
**Pull request by mklzl**
Improve the browser path to the home page

 http://localhost:12345/dolphinscheduler/ui   ====>    http://localhost:12345/dolphinscheduler/ui/

### CODE REVIEW

File: docs/docs/zh/guide/installation/standalone.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ bash ./bin/dolphinscheduler-daemon.sh start standalone-server
 
 ### 登录 DolphinScheduler
 
-浏览器访问地址 http://localhost:12345/dolphinscheduler/ui 即可登录系统UI。默认的用户名和密码是 **admin/dolphinscheduler123**
+浏览器访问地址 http://localhost:12345/dolphinscheduler/ui/ 即可登录系统UI。默认的用户名和密码是 **admin/dolphinscheduler123**
```
Comment (EricGao888) [inline docs/docs/zh/guide/installation/standalone.md:30]: @mklzl  Could u plz update the English docs to keep it consistent? Thx https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/installation/standalone.md

Comment (mklzl) [reply docs/docs/zh/guide/installation/standalone.md:30]: ok

**Comment by EricGao888** [comment]
@mklzl May I ask whether you are still working on this PR? Thanks.

**Comment by zhongjiajie** [comment]
should be combine into #11312

**Pull request closed without merging**